python/ray/serve/_private/constants.py

```diff
@@ -16,19 +16,21 @@
 SERVE_NAMESPACE = "serve"
 
 #: HTTP Address
-DEFAULT_HTTP_ADDRESS = "http://127.0.0.1:8000"
+DEFAULT_HTTP_ADDRESS = os.environ.get(
+    "RAY_SERVE_DEFAULT_HTTP_ADDRESS", "http://127.0.0.1:8000"
+)
```

Comment:
can you check what this one is used for and why it's different from the two below?

Reply:
@GeneDer can you follow up on this one? going to merge the PR

Reply:
Oh interesting this one is actually not used anywhere. Let me follow up to just remove it